### ISSUE TITLE
SmartThings Classic App - Samsung Disabled

### ISSUE THREAD

**Issue by wisc755** [tonesto7/homebridge-smartthings#392]
Hi there,

I was trying to set this up tonight but when I go into the SmartThings Classic App Samsung is preventing me from using it because my location was migrated to the new version.  Any thoughts on how to navigate around this?  

Thanks.

**Comment by neiloakley** [comment]
Just use the new app. Works fine.

**Comment by jvasa** [comment]
The classic app is disabled now, so how do I get the following info from the installation guide:

**app_url** & **app_id** & **access_token** Required To get this information, open the SmartThings HomebridgeV2 SmartApp in your SmartThings Classic Mobile App, and tap on View Configuration Data for Homebridge Notice: The app_url in the example will be different for you.

**Comment by jvasa** [comment]
oh, nevermind, I figured it out. In the new app, I was able to go to HomeBridge v2 under SmartApps and click on "Render the platform data for Homebridge config.json". I was able to get that info there and save it in my config.json.

**Comment by lawiltshire** [comment]
Hi, I am new to this smart home stuff.

I installed homebridge yesterday and got all my devices working apart from SmartThings. I followed the instructions however the smart things classic app isn’t available on the App Store and the new version doesn’t allow me to install the homebridge smart app.

how can I get around this or is it no longer possible 

thanks

**Comment by tomasikp** [comment]
Yes, how do we use the new Samsung Smartthings app?

**Comment by burakcankose** [comment]
Hello, I am having the same problem. I just bought the device but cannot add it to Homebridge. Can a new guide be published? Or the video would be great.

**Comment by Henke30** [comment]
I have the same problem with the new app, the homebridge doesn't show under apps, nor can I install this. Is there a work around for this? is this an area issue perhaps?

**Comment by Henke30** [comment]
> I have the same problem with the new app, the homebridge doesn't show under apps, nor can I install this. Is there a work around for this? is this an area issue perhaps?

Also, does one need to have a SmartThings hub for this to work? Could not find an answer to this in the above closed issue.

**Comment by MB-Jan** [comment]
+1 I would also like to know if this works without a SmartThings hub.

**Comment by brightshado** [comment]
> Update: It does work without the SmartThings hub and with the new SmartThings app on iOS.

Hi, could you please clarify how to add the Homebridge SmartApp in the new iOS app? I do not see this as an option on the SmartApps list. Is it only available to users who had this previously on the Classic app and transferred settings over?

**Comment by mikedmeyer** [comment]
+1 there’s no “My apps” under Add SmartApp...

**Comment by MB-Jan** [comment]
![A68E82EA-4F34-4A7D-8726-26B2E1BE3824](https://user-images.githubusercontent.com/17821126/111858737-d3a24600-896d-11eb-8b4e-c31cd4590f6c.jpeg)
![1071692E-0AA0-413F-BEAC-AF21B6230A4F](https://user-images.githubusercontent.com/17821126/111858739-d735cd00-896d-11eb-96b2-d52f30559473.jpeg)
It 
![5CCD3236-50A0-4140-977B-342DA13A89FC](https://user-images.githubusercontent.com/17821126/111858741-d7ce6380-896d-11eb-9e28-4d8fa3776a49.png)

It really worked without issues on my end on iOS. Above are screenshots that might help you.

**Comment by MB-Jan** [comment]
> > Update: It does work without the SmartThings hub and with the new SmartThings app on iOS.
> 
> Hi, could you please clarify how to add the Homebridge SmartApp in the new iOS app? I do not see this as an option on the SmartApps list. Is it only available to users who had this previously on the Classic app and transferred settings over?

It did not use any other SmartThings app prior setting this up.

**Comment by mikedmeyer** [comment]
Yeah I got it to show up when I did the so-called manual install, but with the “community installer” I had no luck.

**Comment by brightshado** [comment]
> > > Update: It does work without the SmartThings hub and with the new SmartThings app on iOS.
> > 
> > 
> > Hi, could you please clarify how to add the Homebridge SmartApp in the new iOS app? I do not see this as an option on the SmartApps list. Is it only available to users who had this previously on the Classic app and transferred settings over?
> 
> I did not use any other SmartThings app prior setting this up.

Thanks for the reply and screenshots. What you have is definitely the end result I’m trying to get but still unclear on how to add the Homebridge app. If I go to “add smart app” it’s not listed. The only smart app installed for me is one(virtual device creator) that I had installed on the classic version of the app before it was shut down. Would you mind showing how you installed Homebridge on the new version?
![DDAC3BF9-BA5B-4E57-9CB9-364F96B6D11B](https://user-images.githubusercontent.com/79808993/111859652-c4dd7400-8918-11eb-97e7-f9d8a97fbb37.jpeg)
![7DA9EEB8-37C2-41CE-99EB-8FF65B39C2F4](https://user-images.githubusercontent.com/79808993/111859655-c870fb00-8918-11eb-9232-924c151117da.png)

**Comment by brightshado** [comment]
> > > > Update: It does work without the SmartThings hub and with the new SmartThings app on iOS.
> > > 
> > > 
> > > Hi, could you please clarify how to add the Homebridge SmartApp in the new iOS app? I do not see this as an option on the SmartApps list. Is it only available to users who had this previously on the Classic app and transferred settings over?
> > 
> > 
> > I did not use any other SmartThings app prior setting this up.
> 
> Thanks for the reply and screenshots. What you have is definitely the end result I’m trying to get but still unclear on how to add the Homebridge app. If I go to “add smart app” it’s not listed. The only smart app installed for me is one(virtual device creator) that I had installed on the classic version of the app before it was shut down. Would you mind showing how you installed Homebridge on the new version?

Here is what I see
![CBB5ED83-4CBA-4B52-A46A-E36C015F477B](https://user-images.githubusercontent.com/79808993/111859742-6664c580-8919-11eb-8359-7c37cc271202.png)

**Comment by MB-Jan** [comment]
Can you send a screenshot of your SmartThings IDE setup? Then I can compare if there is any difference to my setup.
Maybe it is important: I did not use the community installer. I created a new SmartThings IDE account and used that same Samsung account in the SmartThings app.

**Comment by martonky** [comment]
Hello, I tried everything but I can t rich solution for that. I am using iOS 14.6 and last update og app SmartThing. When I am trying to find Smart app on iOS app, I am not sucessfull. What do you think, what can be wrong? Based on my conclusion, on SmartThing IDE is everything OK.

<img width="1417" alt="Snímka obrazovky 2021-06-22 o 7 14 31" src="https://user-images.githubusercontent.com/48962610/122867769-4e427f00-d32a-11eb-9323-ee31d50e0748.png">

![IMG_4409](https://user-images.githubusercontent.com/48962610/122867790-55698d00-d32a-11eb-8700-b7c22021b83e.PNG)

![IMG_4410](https://user-images.githubusercontent.com/48962610/122867804-5dc1c800-d32a-11eb-8ed5-b72fd345848a.PNG)

**Comment by effektsvk** [comment]
Hi, I have SmartThings version 1.6.68 and the sidebar that @MB-Jan had is missing. I'm even missing the button that @martonky has.

Although, @martonky, did you use this link to create your app? It's the EU region. (https://graph.api.smartthings.com/ is US) https://graph-eu01-euwest1.api.smartthings.com/
